modules/social_features/social_content_block/templates/social-content-block.html.twig

```diff
@@ -0,0 +1,18 @@
+{#
+/**
+ * @file
+ * Custom theme implementation to display a custom content block.
+ */
+#}
+<h4 class="card__title">
+    {{ title }}
+    <div class="card__subtitle">
+        {{ subtitle }}
+    </div>
+</h4>
+{{ topics }}
+<footer class="card__actionbar">
```

Comment:
I'm not sure if really needed leave empty wrapper when a link will be absent.

Reply:
I can set the if link a bit higher, so it wont render when empty.